Makefile

```diff
@@ -9,7 +9,7 @@ cidocker = docker run ${ciargs} ${dockerargs} golang:1.4
 
 all: ${build_image}.created build check integration-tests
```

Comment:
Oh, I missed "all" was used as `all` and not as "all targets" ...
Yeah, that's probably what I need.

Thanks.